### ISSUE TITLE
[NPC][Classic] Gnarlpine Shamans

### ISSUE THREAD

**Issue by Jcb911** [dalaranwow/dalaran-wow#5252]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Gnarlpine Shamans Healing Wave has one of two things; has little to no cooldown or an extremely tiny mana cost (or maybe the mobs have much more mana than they did on retail)

**Expected behaviour**: I understand Relics of Wakening is arguably one of the hardest quests in a races starting zone and it is encouraged to be in a group. Unfortunately, you cannot whisper anyone or ask for help in any chat prior to level 10, therefore you are more than likely going to be doing this quest solo. 
At most, the Shamans should be casting their healing wave spell twice, three times permitted if their mana regens enough throughout the fight. At the moment they are casting it indefinitely as it drains an extremely tiny amount of their mana. At the current state, it is next to impossible to complete this quest if you are doing it solo

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
